hudi-common/src/test/java/org/apache/hudi/common/model/TestOverwriteWithLatestAvroPayload.java

```diff
@@ -72,6 +72,17 @@ public void testActiveRecords() throws IOException {
 
     assertEquals(payload1.combineAndGetUpdateValue(record2, schema).get(), record1);
     assertEquals(payload2.combineAndGetUpdateValue(record1, schema).get(), record2);
+
+    GenericRecord record3 = new GenericData.Record(schema);
+    record3.put("id", "3");
+    record3.put("partition", "partition2");
```

Comment:
Test case for this scenario should have ts as 1L only

Reply:
`ts` is actually not the `preCombine` field, the `preCombine` was passed explicitly.